iceoryx_posh/source/popo/ports/publisher_port_roudi.cpp

```diff
@@ -57,10 +57,7 @@ cxx::optional<capro::CaproMessage> PublisherPortRouDi::tryGetCaProMessage() noex
 
         const auto historyCapacity = m_chunkSender.getHistoryCapacity();
         caproMessage.m_historyCapacity = historyCapacity;
-
-        // provide additional AUTOSAR Adaptive like information
-        caproMessage.m_subType =
-            (0u < historyCapacity) ? capro::CaproMessageSubType::FIELD : capro::CaproMessageSubType::EVENT;
```

Comment:
Since the history size is still part of the capro message, this distinction could be made in an upper layer. Currently this is not used anywhere in iceoryx, at least not seriously

Reply:
You mean that's the task of someone writing an `ara::com` binding?

Reply:
@mossmaurice exactly. Iceoryx should be agnostic and only provide the features to be able to implement such a binding

Reply:
Do you need another example or extend an existing example with the code above?

Reply:
I don't know of any example showing how to use the `InterfacePort`'s. Given the current time constraints, I would keep it as is and just document it in the changelog